apps/worker/src/app/workflow/usecases/send-message/send-message-chat.usecase.ts

```diff
@@ -137,10 +137,11 @@ export class SendMessageChat extends SendMessageBase {
   private async sendChannelMessage(
     command: SendMessageCommand,
     subscriberChannel: IChannelSettings,
-    chatChannel,
+    chatChannel: NotificationStepEntity,
     content: string
   ) {
     const integration = await this.getIntegration({
+      id: subscriberChannel._integrationId,
```

Comment:
When we do send the chat message we are iterating over the subscriber channels and for each channel we do send the message. Here we try to find the integration by the `integrationId` from the subscriber channels array. The subscriber channel array stores the webhook URLs where the message is sent.